publishing.qmd

```diff
@@ -1,5 +1,319 @@
+Your output files are sent to Netlify and Netlify serves them at a URL of the form `{ random identifier }.netlify.app`.
+
+You can manage your deployed site via Netlify's website, and Netlify includes some neat features like one-click rollbacks to a previously deployed site, and the ability to use a custom domain.
+Netlify's free plans limit how many sites you can have and how much bandwidth is used to serve them.
+However, these limits will likely not be reached for low-traffic personal sites.
+
+Rather than using `quarto publish` you can get Netlify to deploy from Git provider.
+In this workflow, you commit your rendered document or site to your repsoitory in a folder (e.g. `docs/` or `_site`) and connect Netlify to your Git provider, setting the folder as the **Publish** directory.
+You can read more about this on the Quarto documentation website at [Publish from a Git Provider](https://quarto.org/docs/publishing/netlify.html#publish-from-git-provider).
+
+Netlify excels in continuous integration and you can set it up to preview deployments based on branches or pull requests.
+The features require a more complicated publishing set up you'll touch on in @sec-ci.
+
+### Other Providers
+
+You are most likely to use Posit Connect or Confluence in organizational settings where your IT department will manage them.
+Hugging Face has a more complicated set up and probably only makes sense if you are using Hugging Face beyond Quarto already.
+
+The `quarto publish` model described in @sec-publishing-process also applies to these providers.
+You can read more about the details of publishing in the [Publishing Guide](https://quarto.org/docs/publishing/) on the Quarto documentation website.
+
+## Sharing Documents in Other Ways {#sec-sharing-other-ways}
+
+You can share self-contained output files like PDFs or MS Word documents in any usual way: by email, placing on a shared drive, or using cloud storage.
+You can do the same with other output files but you need to be careful to also share any dependencies of those files.
+
+For HTML output you have two options: create a self-contained file, or share a directory.
+
+### Create a self-contained file
+
+For HTML output one option is to set the `embed-resources` option to `true`:
+
+``` {.yaml filename="document.qmd"}
+---
+title: Self Contained HTML
+format: 
+  html:
+    embed-resources: true
+---
+```
+
+When rendered this ensures all required dependencies are embedded directly in output `.html` file.
+You can then send, or otherwise share this one file (e.g. `document.html`) and it will appear the same to your viewers as it does to you.
+
+If your document contains math, and you might view it offline, add `self-contained-math: true` to ensure the math libraries are also embedded (by default they aren't because they are quite large):
+
+``` {.yaml filename="document.qmd"}
+---
+title: Self Contained HTML
+format: 
+  html:
+    embed-resources: true
+    self-contained-math: true
```

Comment:
What a bummer that `self-contained` is deprecated but `self-contained-math` isn't `embed-math-resources`, or something like that...